tests/garage/tf/regressors/test_bernoulli_mlp_regressor_with_model.py

```diff
@@ -0,0 +1,235 @@
+import pickle
+from unittest import mock
+
+from nose2.tools.params import params
+import numpy as np
+import tensorflow as tf
+
+from garage.tf.optimizers import ConjugateGradientOptimizer, LbfgsOptimizer
+from garage.tf.regressors import BernoulliMLPRegressorWithModel
+from tests.fixtures import TfGraphTestCase
+
+
+def get_labels(input_shape, xs, output_dim):
+    if input_shape == (1, ):
+        label = [0, 0]
+        # [0, 1] if sign is positive else [1, 0]
+        ys = 0 if np.sin(xs) <= 0 else 1
+        label[ys] = 1
+
```

Comment:
It looks really weird to have a blank line here, since the function reads easily with the indentation.
Please delete the blank line.